bundle/manifests/elasticsearch-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,362 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
The file name should include the operator channel version in it: 
`elasticsearch-operator.v4.7.0.clusterserviceversion.yaml`